test/src/element/testProtocolElementSelectors.js

```diff
@@ -184,18 +184,11 @@ describe('test protocol element selectors', function() {
       .elementsByTag();
 
     Nightwatch.api()
-      .elements('css selector', {selector: '.nock', locateStrategy: 'css selector'}, function callback(result) {
```

Comment:
Any particular reason we had removed these tests?

Reply:
I will add them back, can't seem to remember the reason for which it was removed.